src/screens/errors/NetworkBoundary.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { Flex, Header, Text, Container } from "@urkellabs/ucl";
+
+import styled from "styled-components";
+
+// Components
+import NetworkSwitcher from "components/shared/NetworkSwitcher";
+
+const Center = styled(Flex)`
+  height: 100vh;
+  width: 100%;
+  padding: 50px;
+`;
+
+const NetworkSwitcherWrapper = styled.div`
+  width: 100%;
+  max-width: 500px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+`;
+
+export default class NetworkBoundary extends React.Component {
```

Comment:
I see there's currently no react-baked hook for componentDidCatch, I presume that's why you opted for the class component instead of a functional one, eh?

Reply:
Yeah unfortunately this is the only place where you still are forced to used class components.